### PR TITLE
[taskcluster] Disable Chrome sandboxing in Docker

### LINKED ISSUE #20133: Chrome crashes on Community-TC
We started Taskcluster migration (https://bugzilla.mozilla.org/show_bug.cgi?id=1574668) and now we are seeing widespread Chrome crashes on the new Community-TC instance.

Example: https://community-tc.services.mozilla.com/tasks/aMcaEBP4RIKa_EbrE30xJw/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FaMcaEBP4RIKa_EbrE30xJw%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L550

This [PR](https://github.com/web-platform-tests/wpt/pull/19570) has both Taskcluster and Community-TC jobs. Note that the crash only happens on Community-TC. After checking a few other jobs, it seems the crash is widespread on Community-TC.

### PR DESCRIPTION
**Pull request by Hexcles**
Whenever we see `TASKCLUSTER_ROOT_URL` in the entry point (`runl.py`), we
disable Chrome sandboxing with --no-sandbox.

tox.ini needs to be modified to allow passing of `TASKCLUSTER_ROOT_URL`.

Fix #20133

(Alternative to #20136 )

### CODE REVIEW

**Review by jgraham — DISMISSED** [review]

File: tools/wpt/tests/test_wpt.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def test_run_failing_test():
     failing_test = "/infrastructure/expected-fail/failing-test.html"
 
     assert os.path.isfile("../../%s" % failing_test)
+    assert os.getenv("TASKCLUSTER_ROOT_URL")
```
Comment (Hexcles) [inline tools/wpt/tests/test_wpt.py]: This is expected to fail outside of Taskcluster and will need to be reverted.

However, it currently also fails on Taskcluster, which suggests that `TASKCLUSTER_ROOT_URL` isn't passed in.

File: tools/tox.ini [hunk 2]
```diff
@@ -14,6 +14,7 @@ commands = pytest {posargs}
 passenv =
   HYPOTHESIS_PROFILE
   PY_COLORS
+  TASKCLUSTER_ROOT_URL
```
Comment (Hexcles) [inline tools/tox.ini]: https://tox.readthedocs.io/en/latest/example/basic.html#passing-down-environment-variables

**Review by Hexcles — COMMENTED** [review]
`infrastructure/ tests` now pass, which suggests our change in `run.py` is working as intended.

However, I don't know why `tools/wpt/ tests` still fail. See my inline comments.

**Comment by Hexcles** [comment]
Turns out I was looking at the wrong `tox.ini`. Thanks, @jgraham !

**Comment by Hexcles** [comment]
cc @stephenmcgruer this should hopefully pass everything and I've also dropped the debug commits.

I need to shoot off now. In case this still fails, could you take a look and talk to James? It'd really be great to have this merged before tomorrow so that we can finish our migration before the deadline.

**Comment by stephenmcgruer** [comment]
Ack, will monitor how it goes.

**Review by stephenmcgruer — APPROVED** [review]

**Pull request merged**